ChangeLog

```diff
@@ -89,12 +89,10 @@
  * #926 (Covariance model active parameter set behavior)
  * #928 (Covariance model/ Field/Process properties naming)
  * #932 (Optim algo iteration/evaluation number)
- * #933 (Compilation failed when Muparser is missing)
  * #937 (Small bugs in FunctionalChaosResult::get{Residuals, RelativeErrors})
  * #938 (DistributionFactory::buildEstimator to handle Exception)
  * #939 (The help of the getMaximumDegreeStrataIndex method is wrong.)
  * #941 (Duplicate with SobolIndicesExperiment)
- * #943 (Sample don't show up correctly in notebooks)
```

Comment:
Is this change intentional (removal of two bugs)?

Reply:
Yes, these two bugs affected the git version only, not a published version, so the end user does not care about that.